locales/en.json

```
@@ -0,0 +1,93 @@
+{
+    "locale": {
+        "code": "en",
+        "displayName": "English",
+        "nativeDisplayName": "English",
+        "emoji": "🇺🇸"
+    },
+    "common": {
+        "documentation": "Documentation",
+        "loading": "Loading",
+        "download": "Download",
+        "releases": "Releases",
+        "allVersions": "All Versions",
+        "sourceCode": "Source Code",
+        "faq": "FAQ",
+        "error": "Error",
+        "contributors": "Contributors"
+    },
+    "errors": {
+        "failedToLoadLatestVersion": "Failed to load the latest version. Please try again later."
+    },
+    "downloads": {
+        "macOS": "macOS",
+        "linux": "Linux",
+        "windowsSetup": "Windows Setup",
+        "windowsStandalone": "Windows Standalone",
+        "unsuportedOS": "Unsupported OS"
+    },
+    "home": {
+        "productPage": "Product Page",
+        "alcomDescription": "A fast and open-source VRChat Creator Companion alternative",
+        "cliDescription": "A fast and open-source VRChat Package Manager alternative"
+    },
+    "alcom": {
+        "hero": "A fast open-source alternative of VRChat Creator Companion",
+        "faq": {
+            "fullName": {
+                "question": "What does ALCOM stand for?",
+                "answer": "ALCOM is the complete name of the application, not an acronym."
+            },
+            "nameOrigin": {
+                "question": "Where did the word ALCOM come from?",
+                "answer": "ALCOM was originally derived from the term <b>AL</b>ternative (VRChat Creator) <b>COM</b>panion.<br/>However, <code>alternative companion</code> isn't the name of this tool; it's a description of its purpose."
+            },
+            "vrcGetRelationship": {
+                "question": "What's the relationship with vrc-get-gui and vrc-get?",
+                "answer": "ALCOM is a graphical user interface designed for managing VRChat packages. It's built on top of the open-source command-line tool <a href=\"/cli\">vrc-get</a>. Previously known as <code>vrc-get-gui</code>, ALCOM offers a more user-friendly way to install, update, and manage VRChat assets."
+            }
+        }
+    },
+    "cli": {
+        "hero": "A fast and open-source VRChat Package Manager alternative",
```

```suggestion
        "hero": "A fast and open-source command line client of VRChat Package Manage.",
```

---

Sorry "manage" was typo of "manager" and trailing . is also misss. (removed in worktree but forget to fix in suggestion on github.

```suggestion
        "hero": "A fast and open-source command line client of VRChat Package Manager",
```

is what I intend